samples/data-sealing/common/dispatcher.cpp

```diff
@@ -2,382 +2,113 @@
 // Licensed under the MIT License.
 
 #include "dispatcher.h"
+#include <openenclave/corelibc/stdlib.h>
 #include <stdio.h>
 #include <string.h>
 #include "common.h"
 
-ecall_dispatcher::ecall_dispatcher()
-{
-    m_data = NULL;
-    m_data_size = 0;
-    m_sealed_data = NULL;
-    init_mbedtls();
-}
-
-ecall_dispatcher::~ecall_dispatcher()
-{
-    cleanup_mbedtls();
-}
-
-void ecall_dispatcher::init_mbedtls()
-{
-    const char pers[] = "random data string";
-    mbedtls_entropy_init(&m_entropy_context);
-    mbedtls_ctr_drbg_init(&m_ctr_drbg_contex);
-
-    // mbedtls_ctr_drbg_seed seeds and sets up the CTR_DRBG entropy source for
-    // future reseeds.
-    mbedtls_ctr_drbg_seed(
-        &m_ctr_drbg_contex,
-        mbedtls_entropy_func,
-        &m_entropy_context,
-        (unsigned char*)pers,
-        sizeof(pers));
-}
-
-void ecall_dispatcher::cleanup_mbedtls(void)
-{
-    mbedtls_entropy_free(&m_entropy_context);
-    mbedtls_ctr_drbg_free(&m_ctr_drbg_contex);
-}
-
 int ecall_dispatcher::seal_data(
     int seal_policy,
-    unsigned char* opt_mgs,
+    const unsigned char* opt_msg,
```

Comment:
nit: avoid abbreviations like "opt" and "msg".  Yes I know it was already bad before, so ok to address in a separate PR.